.vscode/settings.json

```diff
@@ -5,15 +5,12 @@
   "editor.formatOnSaveMode": "file",
   "editor.tabCompletion": "on",
   "editor.tabSize": 2,
-  "editor.formatOnSave": true,
+  "editor.formatOnSave": false,
```

Comment:
note: this is now deprecated and `editor.codeActionsOnSave` is recommended for ESLint plugin